.osparc/jupyter-math/runtime.yml

```diff
@@ -22,4 +22,14 @@ callbacks-mapping:
   inactivity:
     service: container
     command: ["python", "/usr/local/bin/service-monitor/activity.py"]
-    timeout: 1
\ No newline at end of file
+    timeout: 1
+compose-spec:                                                                                                                                       
+  version: "3.7"    
+  services:                                                                                                                                         
+    jupyter-math:
+      image: $${SIMCORE_REGISTRY}/simcore/services/dynamic/jupyter-math:$${SERVICE_VERSION}  
+      environment:                                                                                                                                  
+        - OSPARC_API_HOST=$${OSPARC_VARIABLE_API_HOST}                                                                                                
```

Comment:
This does not exist in Osparc codebase ad it will not resolve on all deployments

Reply:
Do you mean this is because the API server is not available on some deployments? Or just because the variable is not set yet?

@pcrespov in your new PR on simcore, what will happen if there is no API server is available? Will this var be empty/None or so, or will it just not be set?

Reply:
Actually, waiting for you review ;-)  https://github.com/ITISFoundation/osparc-simcore/pull/5695

Reply:
@wvangeit i will then pull it and use it locally.

Reply:
> what will happen if there is no API server is available? Will this var be empty/None or so, or will it just not be set?

Good question. i will double check.

Reply:
Missed your PR. Got distracted. It's all good now.

Reply:
I would vote for empty string. Then the user and the python api client can catch this, and notice that the api host is undefined. (in a sense some 'null' instead of empty string would be better, but not sure if there is a stable null equivalent in the world of env vars)

Reply:
I prefer that `OSPARC_VARIABLE_*` have to be resolvable in order for the service to start. Unlike the `OSPARC_VARIABLE_VENDOR_SECRET_*`, which are define din the db, these are provided always by the platform.

If you are requesting an `OSPARC_VARIABLE_*`, which does not exist, this is a clear way to signal that you did something wrong.

Also I think it's the user's responsibility to make sure that the service stars once they make changes to it. He can always test it in a local deployment before publishing it elsewhere. Unfortunately we don't have a better workflow for this part.

Reply:
- FYI vendor secrets are also provided by the service owner.
- directorv2 **also** fails to start service when the vendor secrets is not available.